gripper/firmware/interfaces_z_motor.cpp

```diff
@@ -80,10 +80,13 @@ static tmc2130::configs::TMC2130DriverConfig MotorDriverConfigurations{
                                .tbl = 0x2,
                                .mres = 0x4},
                   .coolconf = {.sgt = 0x6},
-                  .pwmconf = {.freewheel = 0x2}},
+                  .pwmconf = {.pwm_ampl = 0x80,
```

Comment:
can we add what this does or a quick description of the issue it fixes to the PR description and eventual commit body